src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_aks_commands.py

```diff
@@ -9213,9 +9213,11 @@ def test_aks_create_with_azurekeyvaultkms_private_key_vault(self, resource_group
         ]).get_output_in_json()
 
         create_cmd = 'aks create --resource-group={resource_group} --name={name} ' \
+                     '--aks-custom-headers AKSHTTPCustomFeatures=Microsoft.ContainerService/EnableAPIServerVnetIntegrationPreview ' \
                      '--assign-identity {identity_id} ' \
                      '--enable-azure-keyvault-kms --azure-keyvault-kms-key-id={key_id} ' \
                      '--azure-keyvault-kms-key-vault-network-access=Private --azure-keyvault-kms-key-vault-resource-id {kv_resource_id} ' \
+                     '--enable-apiserver-vnet-integration=true ' \
```

Comment:
This is an action option and does not accept a value.

Reply:
done